### ISSUE TITLE
We need to pin this as a project so people find it easily

### ISSUE THREAD

**Issue by chagai95** [Trustroots/community#46]

**Comment by Lemoert** [comment]
@chagai95 - Closing to clean up the issue list a bit. Let me know if there's anything you want to add to the projects :) /Martin A.

**Comment by chagai95** [comment]
where should I put this todo then?

**Comment by Lemoert** [comment]
@chagai95 - Not sure what you wanted to pin or what people need to find easily :)

**Comment by chagai95** [comment]
The community repository on the main GitHub page - on github.com/trustroots there are 3 other pinned repos and many inactive repos in a list it'll be nice if the community repo could also be pinned

**Comment by Lemoert** [comment]
Right, makes sense now! Reopening issue and will ask Guaka to pin it once [we've decided on a new name](https://github.com/Trustroots/community/issues/31).
